src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -70,6 +71,12 @@ public class ShardRangeAssignment
             public int Shard;
         }
 
+        public Dictionary<int, BucketMigration> BucketMigrations;
```

Comment:
This allows concurrent migrations?

Reply:
Yes, having one at a time it is only a matter of policy. 
(currently it is blocked to have concurrent migrations, but we can unblock it)